coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You need to update your `.gitignore` file to exclude `/coverage`. 

See [All the badges](https://learn.launchacademy.com/teams/spring-2016/curricula/onsite/lessons/all-the-badges), under Coveralls, Step 3.